packages/unleash-docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:5.8
+
+COPY . .
+
+RUN npm install --production && \
+    mkdir migrations && cp -r node_modules/unleash-server/migrations/* migrations/.
```

Comment:
this i really did not like. but because it is a published artifact you can not just refer migrations inside it (because they again refer to "scripts/migration-runner.js".